### PR TITLE
Proofread Credential Format Profiles

### PR DESCRIPTION
**Pull request by selfissued**
Correct issues identified while proofreading the Credential Format Profiles appendix.

(Yes, I'm fully aware that every change here is a merge conflict with #171, due to it moving the section.  I'll resolve the merge conflicts after #171 is merged.)

### CODE REVIEW

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -2006,19 +2006,19 @@ Wallet Providers may also provide a market place where Issuers can register to b
 
 # Credential Format Profiles {#format_profiles}
 
-This specification defines several extension points to accommodate the differences across Credential formats. Sets of Credential format specific parameters or claims referred to as Credential format Identifiers are identified by the Credential format identifier and used at each extension point.
+This specification defines several extension points to accommodate the differences across Credential formats. Sets of Credential format specific parameters or claims referred to as Credential format Identifiers are identified by the Credential format identifier and used at these extension points.
```
Comment (peppelinux) [inline openid-4-verifiable-credential-issuance-1_0.md:2009]:
```suggestion
This specification defines several extension points to accommodate the differences across Credential formats. Sets of Credential format specific parameters or claims referred to as Credential format identifiers are identified by the Credential format identifier and used at these extension points.
```

**Review by peppelinux — APPROVED** [review]

**Comment by selfissued** [comment]
I applied the `credentials_supported` -> `credential_configurations_supported` correction suggested by @TakahikoKawasaki in #172.

**Review by Sakurann — APPROVED** [review]
IMO in majority of the places the current text reads ok, but I won't die on this hill.

**Review by tplooker — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 2]
```diff
@@ -2085,44 +2085,44 @@ The following is a non-normative example of a Credential Response with Credentia
 
 <{{examples/credential_response_jwt_vc_json.txt}}
 
-### VC Secured using Data Integrity, using JSON-LD, with Proof Suite Requiring Linked Data Canonicalization
+### VC Secured using Data Integrity, using JSON-LD, with a Proof Suite Requiring Linked Data Canonicalization
```
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:2088]:
```suggestion
### VC Secured using Data Integrity, using JSON-LD, with a Proof Suite requiring Linked Data Canonicalization
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 3]
```diff
@@ -2162,9 +2162,9 @@ The following is a non-normative example of a Credential Response with Credentia
 
 The Credential format identifier is `jwt_vc_json-ld`.
 
-When the `format` value is `jwt_vc_json-ld`, entire Credential Offer, Authorization Details, Credential Request and Credential Issuer metadata, including `credential_definition` object, MUST NOT be processed using JSON-LD rules.
+When the `format` value is `jwt_vc_json-ld`, the entire Credential Offer, Authorization Details, Credential Request and Credential Issuer metadata, including `credential_definition` object, MUST NOT be processed using JSON-LD rules.
 
-`@context` value in the `credential_definition` object could be used by the Wallet to check whether it supports a certain VC or not. If necessary, the Wallet could apply JSON-LD processing to the Credential issued by the Credential Issuer.
+The `@context` value in the `credential_definition` object can be used by the Wallet to check whether it supports a certain VC or not. If necessary, the Wallet could apply JSON-LD processing to the Credential issued by the Credential Issuer.
```
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:2167]:
```suggestion
The `@context` value in the `credential_definition` parameter can be used by the Wallet to check whether it supports a certain VC or not. If necessary, the Wallet could apply JSON-LD processing to the Credential issued by the Credential Issuer.
```

**Pull request merged**